packages/atomic/src/components/commerce/atomic-commerce-search-box-recent-queries/atomic-commerce-search-box-recent-queries.mdx

```diff
@@ -0,0 +1,40 @@
+import {Canvas, ArgTypes, Meta, Title, Description, Stories, Subtitle} from '@storybook/blocks';
+import * as AtomicCommerceSearchBoxRecentQueriesStories from './atomic-commerce-search-box-recent-queries.new.stories';
+
+
+<Meta of={AtomicCommerceSearchBoxRecentQueriesStories} />
+
+<Title />
+<Subtitle><Description/></Subtitle>
+
+<Canvas
+  additionalActions={[
+    {
+      title: 'Open in GitHub',
+      onClick: () => {
+        window.open(
+          'https://github.com/coveo/ui-kit/blob/master/packages/atomic/src/components/commerce/atomic-commerce-search-box-recent-queries/atomic-commerce-search-box-recent-queries.ts',
+          '_blank'
+        );
+      },
+    },
+  ]}
+  of={AtomicCommerceSearchBoxRecentQueriesStories.Default}
+/>
+
+## Usage
```

Comment:
Perhaps we should mention somewhere that the component will have no effect when analytics are disabled.